### ISSUE TITLE
Is there a place where I can find a full example of how to use this plugin?

### ISSUE THREAD

**Issue by daveryaaziem** [Wizcorp/phonegap-facebook-plugin#937]
The readme is a little unclear. It says the plugin allows you to use fb sdk just as you would in a web app, but then there are methods for login, etc. When I try to access those methods I get errors. Is there anyway someone can dumb it down for a begginer like myself?

**Comment by lewistaylor** [comment]
Second this, on both ngCordova and the docs here, it's very difficult to see how to properly set this up.

I notice on the code samples for iOS there is a reference to facebookConnectPlugin.js, but no mention in the documentation.

**Comment by marie-dk** [comment]
You should provide the steps you did to install the plugin, and some code so that others can see what you are doing... makes it much easier to help.

**Comment by poppahorse** [comment]
+1

**Comment by byhoratiss** [comment]
+1 for this also!
